cooltools/sandbox/expected_smoothing.py

```diff
@@ -266,40 +225,102 @@ def agg_smooth_cvd(
 
     Returns
     -------
-    cvd_smoothed : pandas.DataFrame
+    cvd : pandas.DataFrame
         A cvd table with extra column for the log-smoothed contact frequencies (by default, "balanced.avg.smoothed").
-    """
 
-    cols = dict(DEFAULT_CVD_COLS, **({} if cols is None else cols))
+    Notes
+    -----
+    Parameters in "cols"
+
+    dist:
+
+    n_pixels:
 
-    if groupby is None:
-        groupby = []
-    elif isinstance(groupby, str):
-        groupby = [groupby]
-    elif isinstance(groupby, Iterable):
-        groupby = list(groupby)
-    else:
-        raise ValueError("groupby must be a string, a list of strings, or None")
+    n_contacts:
 
-    cvd_smoothed = _agg_smooth_cvd(
+    contact_freq:
+
+    """
+    cvd_smoothed = _smooth_cvd_group(
         cvd,
-        groupby=groupby,
         sigma_log10=sigma_log10,
         window_sigma=window_sigma,
         points_per_sigma=points_per_sigma,
         cols=cols,
+        suffix=suffix,
     )
 
-    cvd_smoothed.drop(
-        [cols["n_pixels"], cols["n_contacts"]], axis="columns", inplace=True
+    # add aggeragated smoothed columns to the result
+    cvd = cvd.merge(
+        cvd_smoothed[[cols["output_prefix"] + suffix, cols["dist"]]],
+        on=[cols["dist"]],
+        how="left",
     )
 
-    # cvd = cvd.drop(cols["contact_freq"], axis='columns', errors='ignore')
+    return cvd
 
-    # cvd = cvd.merge(
-    #     cvd_smoothed,
-    #     on=groupby + [cols["dist"]],
-    #     how="left",
-    # )
 
-    return cvd_smoothed
+def per_region_smooth_cvd(
```

Comment:
should per_region_smooth_cvd and genomewide_smooth_cvd migrate into expected.py since the dictionary was migrated?